src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  const isSelected = tab => tab === selectedTab;
```

Comment:
Comparing objects is a bad practice. better compare `tab.id` and `selectedTab.id`

Reply:
> Comparing objects is a bad practice. better compare `tab.id` and `selectedTab.id`

Good point, and the name is not informative, what selected I have no idea